### DISCUSSION TITLE
Could not find TARGET igtlioConverter

### DISCUSSION THREAD

**Discussion by 1939938853** [PlusToolkit/PlusLib#933] [Q&A]
Hello,

I'm trying to build pluslib. When doing the cmake config, I got the following error. What can be the problem? Thanks!

`
CMake Error at src/PlusCommon/CMakeLists.txt:38 (GET_PROPERTY):
  GET_PROPERTY could not find TARGET igtlioConverter.  Perhaps it has not yet
  been created.
`

**Comment by Sunderlandkyl** [comment]
igtlioConverter is built in OpenIGTLinkIO. Are you building PlusLib by itself, or are you using [PlusBuild](https://github.com/PlusToolkit/PlusBuild)?

**Comment by 1939938853** [comment]
I'm trying to build pluslib itself. Building plusBuild no problem, but seems a little bit difficult to control the INSTALL.

**Comment by 1939938853** [comment]
Got it. Need to enable PLUSBUILD_USE_OpenIGTLink.
